pkg/client/auth/oidc/pkce.go

```diff
@@ -89,10 +89,20 @@ func AuthenticatePkce(config PKCEDetails) (*TokenCredentials, error) {
 
 	server := &http.Server{}
 
-	go server.Serve(listener)
+	go func() {
+		err := server.Serve(listener)
+		if err != nil {
+			panic(err)
```

Comment:
i don't think we should introduce panic

Reply:
I guess I thought that if we can't start the http server that would be bad but yea, I can add a log instead of a panic.  I think that makes sense